src/include/homestore/btree/btree.hpp

```diff
@@ -100,6 +127,16 @@ class Btree {
 
     // static void set_io_flip();
     // static void set_error_flip();
+#ifdef _PRERELEASE
+    void set_flip_point(std::string flip) {
+        m_flips.set_flip(flip);
+    }
+    void set_flips(std::vector< std::string > flips) {
+        for (const auto& flip : flips) {
+            set_flip_point(flip);
+        }
+    }
```

Comment:
Please avoid pass-by-value on containers, as it could be very expensive without any hint.